docs/experiments-monot5-tpu.md

```diff
@@ -211,4 +211,61 @@ You should see the same result.
 
 If you were able to replicate these results, please submit a PR adding to the replication log! Please mention in your PR if you find any difference!
 
+## Train monoT5
+
+First, download the MS MARCO train triples:
+```
+wget https://storage.googleapis.com/duobert_git/triples.train.small.tar.gz
+tar -xvf triples.train.small.tar.gz
+rm triples.train.small.tar.gz
+```
+
+Then convert the train triples to t5 input format:
+```
+python -m pygaggle.data.create_msmarco_t5_training_pairs --triples_train triples.train.small.tsv --output_to_t5 query_doc_pairs.train.tsv
+```
+
+Next, copy the input file to Google Storage. TPU training will read data directly from `gs`
+```
+gsutil cp query_doc_pairs.train.tsv ${GS_FOLDER}/query_doc_pairs.train.tsv
+```
+
+Recall the environment variables
+```
+export MODEL=<t5 pretrain model, e.g. base, 3B>
+export GS_FOLDER=<gs folder to store checkpoints>
+export PROJECT_NAME=<gcloud project name>
+export TPU_NAME=<name of tpu to create>
+```
+
+Copy pre-trained checkpoint to our target model
+```
+echo "model_checkpoint_path: \"model.ckpt-999900\"" > checkpoint
+gsutil cp checkpoint ${GS_FOLDER}
+gsutil cp gs://t5-data/pretrained_models/${MODEL}/model.ckpt-999900* ${GS_FOLDER}
+```
+
+```
+export EXPNO=002
```

Comment:
We don't need this, right?

Reply:
@rodrigonogueira4  is base ckpt=999900 for all models in training? I made it configurable rn, if it is fixed, i'll change it back to hardcode